src/components/Modal/AttestModal.tsx/AttestModal.tsx

```diff
@@ -220,8 +237,9 @@ export const AttestModal: FC<AttestModalProps> = ({
 						<div className='border p-4'>
 							{isLoading ? (
 								<div>Loading user&apos;s organizations</div>
-							) : data && data?.length > 0 ? (
-								data?.map(ao => (
+							) : fetchedOrganisations &&
```

Comment:
`fetchedOrganisations &&` is extra, if `fetchedOrganisations` is undefined, `fetchedOrganisations?.length > 0` will be false already

Reply:
eslint doesn't like that 🤣